File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domene/Sykefraværsstatistikk.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene;

import java.math.BigDecimal;

public interface Sykefraværsstatistikk {

  int getÅrstall();

  int getKvartal();

  int getAntallPersoner();

  BigDecimal getTapteDagsverk();

  BigDecimal getMuligeDagsverk();
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/fellesdomene/Sykefraværsstatistikk.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene

import java.math.BigDecimal

sealed interface Sykefraværsstatistikk {
    val årstall: Int
    val kvartal: Int
    val antallPersoner: Int
    val tapteDagsverk: BigDecimal?
    val muligeDagsverk: BigDecimal?
}
